lib/oauthenticator/version.rb

```diff
@@ -1,5 +1,5 @@
 # OAuthenticator 
 module OAuthenticator
   # OAuthenticator::VERSION
-  VERSION = "1.3.1"
+  VERSION = "1.3.4"
```

Comment:
confusion between 1.3.2 and 1.3.4 came from version in venuenext repo and this one...
I must have amend my commit message but forgot to git add the number change.

anyway, I will remove that commit from this PR, so you can manage the versions outside of PR like you ask